authentication/auth.go

```diff
@@ -745,6 +749,8 @@ func GetDefaultValues() map[string]string {
 	defaults := make(map[string]string)
 	collectionExerciseSid, _ := uuid.NewV4()
 
+	var ID = fmt.Sprintf("%011d", rand.Int63n(1e11))
+
```

Comment:
Not a blocking change, but something you might want to consider (all suggestions are related):
```suggestion
	var PARTICIPANT_ID = "ABC-" + fmt.Sprintf("%011d", rand.Int63n(1e11))

```